### PR TITLE
vim, nvim: update outdated translations

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
Ref: #14181

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kant — APPROVED** [review]
Spanish chapter: LGTM

**Review by kant — APPROVED** [review]
LGTM

File: pages.fr/common/vim.md [hunk 1]
```diff
@@ -18,11 +18,11 @@
 
 - Sauvegarder et fermer :
 
-`:wq<Entrée>`
+`{{ZZ|:wq<Entrée>}}`
 
 - Annuler la dernière opération :
 
-`u`
+`<Esc>u`
```
Comment (noraj) [inline pages.fr/common/vim.md:25]: Inconsistency: some translations have the `Undo the last operation:`, some are missing it.

File: pages.id/common/nvim.md [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 - Menyimpan (write) berkas, dan keluar:
 
-`<Esc>:wq<Enter>`
+`{{<Esc>ZZ|<Esc>:wq<Enter>}}`
```
Comment (noraj) [inline pages.id/common/nvim.md:33]: Inconsistency: On some translations you put `{{<Esc>ZZ|<Esc>:wq<Enter>}}`, while on other you miss the escape sequence `{{ZZ|:wq<Entrée>}}`

File: pages.fr/common/vim.md [hunk 3]
```diff
@@ -18,11 +18,11 @@
 
 - Sauvegarder et fermer :
 
-`:wq<Entrée>`
+`{{ZZ|:wq<Entrée>}}`
```
Comment (noraj) [inline pages.fr/common/vim.md:21]: While adding `ZZ` as an alternative to `:wq`, we should also add `:x`.

**Review by noraj — COMMENTED** [review]
@sebastiaanspeck Thank for your PR, but please don't merge too soon so more people have a chance to review before merging. For example

**Pull request merged**